tests/integration/security/rbac/util/rbac_util.go

```diff
@@ -39,58 +40,56 @@ type TestCase struct {
 // *** For TCP: EOF error
 // * If the policy is allow:
 // *** Response code is 200
-func CheckRBACRequest(tc TestCase) error {
+func (tc TestCase) Check() error {
```

Comment:
Why does this change from `CheckRBACRequest` to `Check`? I don't think it's clear what does `util.Check` does when reading the code. Also, function comment wasn't updated.

Reply:
Because the method is already scoped to the `rbac` package and is only used from `RBAC` tests. It's not useful and somewhat anti-idiomatic to stutter when making type/method names in Go.